src/ign_TEST.cc

```diff
@@ -104,7 +104,7 @@ TEST_F(CmdLine, DETAIL_IGN_UTILS_ADD_DISABLED_PREFIX(list))
 
 //////////////////////////////////////////////////
 /// \brief Check --help message and bash completion script for consistent flags
-TEST(ignTest, GuiHelpVsCompletionFlags)
+TEST(ignTest, IGN_UTILS_TEST_DISABLED_ON_WIN32(GuiHelpVsCompletionFlags))
```

Comment:
If we are not solving it right away I think we should add the main issue with disabled tests to track it as part of the green CI efforts.

```suggestion
// See https://github.com/gazebo-tooling/release-tools/issues/398
TEST(ignTest, IGN_UTILS_TEST_DISABLED_ON_WIN32(GuiHelpVsCompletionFlags))
```